<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2446</width>
    <height>1672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>160</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pristina</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>Original Image </string>
   </property>
  </widget>
  <widget class="QLabel" name="finalLabel">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>160</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pristina</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>OutPut Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>110</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load Image</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="webcamLabel">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>980</y>
     <width>640</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pristina</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>Original Video</string>
   </property>
  </widget>
  <widget class="QLabel" name="webcam2Label">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>980</y>
     <width>640</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pristina</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>OutPut Video</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sobelVideoButton">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>930</y>
     <width>641</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Sobel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>910</y>
     <width>641</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="openButton">
      <property name="text">
       <string>Open Webcam</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Close Webcam</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>711</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Ravie</family>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>                     HIGH PASS FILTERS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cannyButton">
      <property name="text">
       <string>Canny</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="laplacianButton">
      <property name="text">
       <string>Laplacian</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sobelButton">
      <property name="text">
       <string>Sobel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>711</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Ravie</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>                     LOW PASS FILTERS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="boxBlurButton">
      <property name="text">
       <string>Box Blur</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="blurButton">
      <property name="text">
       <string>Blur</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="gaussianBlurButton">
      <property name="text">
       <string>Gaussian Blur</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>711</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ravie</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>840</y>
     <width>711</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ravie</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>Video</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>120</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetButton">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
